src/server/acl/acl_family.cc

```diff
@@ -255,12 +441,21 @@ void AclFamily::Register(dfly::CommandRegistry* registry) {
                    .HFUNC(DelUser);
   *registry << CI{"ACL WHOAMI", CO::ADMIN | CO::NOSCRIPT | CO::LOADING, 1, 0, 0, 0, acl::kWhoAmI}
                    .HFUNC(WhoAmI);
+  *registry << CI{"ACL SAVE", CO::ADMIN | CO::NOSCRIPT | CO::LOADING, 1, 0, 0, 0, acl::kSave}.HFUNC(
+      Save);
+  *registry << CI{"ACL LOAD", CO::ADMIN | CO::NOSCRIPT | CO::LOADING, 1, 0, 0, 0, acl::kLoad}.HFUNC(
+      Load);
 }
 
 #undef HFUNC
 
-void AclFamily::Init(facade::Listener* main_listener) {
+void AclFamily::Init(facade::Listener* main_listener, UserRegistry* registry) {
   main_listener_ = main_listener;
+  registry_ = registry;
+  auto acl_file = absl::GetFlag(FLAGS_aclfile);
+  if (!acl_file.empty()) {
+    Load({}, nullptr);
+  }
```

Comment:
If you don't want to add wrapper functions for this only case you can pass a capturing reply builder into ti

Reply:
But I suggest not to and instead make an error-reply returning internal function 🙂

Reply:
I just overloaded `Load` :)